src/Storage/Storage.Management/help/New-AzDataLakeGen2Item.md

```diff
@@ -0,0 +1,348 @@
+ [-DefaultProfile <IAzureContextContainer>] [-ConcurrentTaskCount <Int32>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **New-AzDataLakeGen2Item** cmdlet creates a file or folder in a container in an Azure storage account.
+This cmdlet only works if Hierarchical Namespace is enabled for the Storage account.
+
+## EXAMPLES
+
+### Example 1: Create a folder with specified Permission, Umask, properties, and metadata
+```
+PS C:\>New-AzDataLakeGen2Item -Container "testcontainer" -Path "dir1/dir2/" -Folder -Permission rwxrwxrwx -Umask ---rw---- -Property @{"CacheControl" = "READ"; "ContentDisposition" = "True"} -Metadata  @{"tag1" = "value1"; "tag2" = "value2" }
+
+   Container Uri: https://storageaccountname.blob.core.windows.net/container1
+
+Name                 IsDirectory  BlobType  Length          ContentType                    LastModified         AccessTier SnapshotTime         IsDeleted  Permissions 
+----                 -----------  --------  ------          -----------                    ------------         ---------- ------------         ---------  ----------- 
+dir1/dir2/           True                                   application/octet-stream       2019-10-14 07:54:47Z                                 False      rwx--xrwx
+```
+
+This command creates a folder with specified Permission, Umask, properties, and metadata
+
+### Example 2: Create(upload) a data lake file from a local source file, and the cmdlet runs in background
+```
+PS C:\> $task = New-AzDataLakeGen2Item  -Container "testcontainer" -Path "dir1/dir2/file1" -Source "c:\sourcefile.txt" -Force -asjob
+PS C:\> $task | Wait-Job
+PS C:\> $task.Output
+
+   Container Uri: https://storageaccountname.blob.core.windows.net/container1
+
+Path                 IsDirectory  Length          ContentType                    LastModified         Permissions  Owner      Group               
+----                 -----------  ------          -----------                    ------------         -----------  -----      -----               
+dir1/dir2/file1      False        14400000        application/octet-stream       2019-10-29 09:04:44Z
+```
+
+This command creates(upload) a data lake file from a local source file, and the cmdlet runs in background.
+
+## PARAMETERS
+
+### -AsJob
+Run cmdlet in the background
+
+```yaml
+Type: System.Management.Automation.SwitchParameter
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ClientTimeoutPerRequest
+The client side maximum execution time for each request in seconds.
+
+```yaml
+Type: System.Nullable`1[System.Int32]
+Parameter Sets: (All)
+Aliases: ClientTimeoutPerRequestInSeconds
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ConcurrentTaskCount
+The total amount of concurrent async tasks.
```

Comment:
The concurrency is on file level or chunk level? Maybe we can explain it more accurately in help.

Reply:
is process level. 
Actually, it's no use in this cmdlet, I will remove it from all no use cmdlets.
It's only useful in the upload/download cmdlets.